stock_release_channel_process_end_time/models/res_config_settings.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, SUPERUSER_ID, _
+from odoo.exceptions import UserError
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    update_scheduled_date = fields.Boolean(
```

Comment:
This config is a bit misleading as the picking scheduled date is anyway updated by the release data + delay.

Reply:
Could be renamed to stock_release_use_channel_end_date